packages/api/src/logger/index.ts

```diff
@@ -350,11 +351,12 @@ interface PrismaLoggingConfig {
  */
 export const handlePrismaLogging = (config: PrismaLoggingConfig): void => {
   const logger = config.logger.child({
-    prisma: { clientVersion: config.db['_clientVersion'] },
+    prisma: { clientVersion: Prisma.prismaVersion },
```

Comment:
Hi @aggmoulik can you describe why you need this change and what it addresses in a different PR since this PR is intended to handle a dev server restart on toml edit not logging.

Also, I have been using v033.3 with Typescript and logging to do see the current Prisma version logged out appropriately.

Thanks!

Reply:
This is the problem I was facing here by the typing of PrismaClient as it was not building and I looked for the better solution then I came around with this.

Reply:
Sorry for not not understanding, but when you say

> the typing of PrismaClient as it was not building

can you provide an example (steps taken, behavior seen, code, screenshot) of the issue and what it was blocking or preventing you from doing and then after this fix how it addressed it (screenshot, example, etc). Thanks!